licenses.yaml

```diff
@@ -5036,21 +4918,15 @@ module: web-console
 license_name: Apache License version 2.0
 copyright: Microsoft Corp.
 version: 1.10.0
-notice: |
-  /*! *****************************************************************************
-  Copyright (c) Microsoft Corporation. All rights reserved.
-  Licensed under the Apache License, Version 2.0 (the "License"); you may not use
-  this file except in compliance with the License. You may obtain a copy of the
-  License at http://www.apache.org/licenses/LICENSE-2.0
 
-  THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
-  WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
-  MERCHANTABLITY OR NON-INFRINGEMENT.
+---
 
-  See the Apache Version 2.0 License for specific language governing permissions
-  and limitations under the License.
-  ***************************************************************************** */
+name: "tslib"
```

Comment:
are there 2 versions of tslib? And does this version not have the `NOTICE` file?

Reply:
So that removed notice was removed because this is an auto generated block. It looks like you added the `notice:` key in https://github.com/apache/incubator-druid/pull/8306/files#diff-66ffa06e81466a207a1dad823ec2fe90R5049 but `script/licenses` which is what makes this block is not aware of it. What is the deal here? What does the `notice:` key represent and should `script/licenses` be updated to generate it?